### ISSUE TITLE
dotnet publish don't copy all dll's

### ISSUE THREAD

**Issue by sidatacom** [dotnet/core#4299]
I created a new .net core wpf application and add a com dependency.

```
<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">

  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>netcoreapp3.1</TargetFramework>
    <UseWPF>true</UseWPF>
  </PropertyGroup>

  <ItemGroup>
    <COMReference Include="Microsoft.Office.Access.dll">
      <Guid>4affc9a0-5f99-101b-af4e-00aa003f0f07</Guid>
      <VersionMajor>9</VersionMajor>
      <VersionMinor>0</VersionMinor>
      <WrapperTool>tlbimp</WrapperTool>
      <Lcid>0</Lcid>
      <Isolated>false</Isolated>
      <Private>true</Private>
    </COMReference>
  </ItemGroup>

</Project>

```
On publishing to folder the COM+ reference dll will not copyed to the publishing folder.

Does exist a event to copy it manually after publishing ?

[WpfApp1.zip](https://github.com/dotnet/core/files/4232081/WpfApp1.zip)

**Comment by scalablecory** [comment]
Is it possible it can't find the DLL? Is there any error? @nguerrera

**Comment by nguerrera** [comment]
Duplicate of https://github.com/dotnet/sdk/issues/3993

**Comment by nguerrera** [comment]
This will be fixed in SDK 3.1.200, which is available in preview now with VS 16.5 previews.

**Comment by sidatacom** [comment]
With preview 16.5 the problem still exists !!!
Now the dll name will corrupted

Interop.Microsoft.Office.Interop.Access.dll;Interop.Microsoft.Office.Interop.Access.Dao.dll;Interop.Microsoft.Office.Core.dll

only one file is copyed with all names inside !!!
